common/buildcraft/core/item/ItemPaintbrush_BC8.java

```diff
@@ -195,7 +195,12 @@ public boolean useOnBlock(World world, BlockPos pos, IBlockState state, Vec3d hi
             if (result == EnumActionResult.SUCCESS) {
                 ParticleUtil.showChangeColour(world, hitPos, colour);
                 SoundUtil.playChangeColour(world, pos, colour);
-                usesLeft--;
+
+                boolean isHolderCreative = player.isCreative();
+                if (!isHolderCreative) {
```

Comment:
Bit minor, but as there's no uses for this, can you inline ````isHolderCreative```` into the if statement?

Reply:
sure